examples/knowledge_tuning/translation_example/indic_trans_server.py

```
@@ -0,0 +1,107 @@
+            {
+                "text": translations[0],
+                "index": 0,
+                "logprobs": None,
+                "finish_reason": "stop",
+            }
+        ],
+    }
+
+
+@app.get("/v1/completions")
+async def translate(source_lang: str, target_lang: str, prompt: str):
```

Pylint (3.3.7)

[error] function already defined line 67

(E0102)